E3Next/Utility/e3util.cs

```diff
@@ -516,6 +516,14 @@ public static void GiveItemOnCursorToTarget(bool moveBackToOriginalLocation = tr
                 e3util.TryMoveToLoc(currentX, currentY);
             }
         }
+        public static bool IsShuttingDown()
+        {
+            if(EventProcessor.CommandList["/shutdown"].queuedEvents.Count > 0)
```

Comment:
is this dict guaranteed to have this key?

Reply:
yes